deploy/operator.yaml

```diff
@@ -23,168 +23,52 @@ rules:
   - apps
   resources:
   - deployments
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - apps
-  resources:
   - statefulsets
   verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
+  - '*'
```

Comment:
is it a good practice to use "*" to acquire all permissions. one of the rule is acquiring only necessary permissions.

Reply:
yes, you are right, but the original permissions are equal '*'.